invenio_files_rest/ext.py

```diff
@@ -92,6 +93,14 @@ def part_factories(self):
             self.app.config.get('FILES_REST_MULTIPART_PART_FACTORIES', [])
         ]
 
+    @cached_property
+    def upload_factories(self):
+        """Factory getting list of webargs schemas for parsing part number."""
+        return [
+            import_string(x) for x in
```

Comment:
I will open separate issue for it, as it's all of the methods in ``ext.py`` that has this problem.